checker/checker.ml

```diff
@@ -178,7 +174,9 @@ let print_usage_channel co command =
   output_string co command;
   output_string co "coqchk options are:\n";
   output_string co
-"  -R dir coqdir          map physical dir to logical coqdir\
+"  -Q dir coqdir          map physical dir to logical coqdir\
+\n  -R dir coqdir          synonymous for -Q\
```

Comment:
in `coqtop`, the doc for `-R` says "map recursively". Why is it different in the checker? Because we don't need to resolve names?

Reply:
I documented it in the reference manual: "recursivity" is irrelevant in `coqchk`. Not to mention that it's actually badly phrased in `coqtop`. Qualified import with `-Q` in `coqtop` is also recursive, the adjective used should rather be *relative* vs. *absolute* (or maybe implicit).